api/managementpb/actions.proto

```diff
@@ -250,6 +250,13 @@ message StartPostgreSQLShowIndexActionResponse {
   string pmm_agent_id = 2;
 }
 
+// MongoDBOptions is used to passing TLS certificates and password.
+message MongoDBOptions {
+  string tls_certificate_key = 1;
+  string tls_certificate_key_file_password = 2;
+  string tls_ca = 3;
+}
```

Comment:
Comments?

And why the are not inline in StartMongoDBExplainActionRequest?

Reply:
good point, will update

Reply:
actually we don't need it at all.

Reply:
Removed